libraries/SITL/SITL.cpp

```diff
@@ -105,6 +105,24 @@ const AP_Param::GroupInfo SIM::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("CAN_SRV_MSK",   29, SIM,  can_servo_mask, 0),
 
+#if HAL_NUM_CAN_IFACES > 0
+    // @Param: CAN_TYPE0
+    // @DisplayName: transport type for first interface
```

Comment:
For the Display name we should probably prefix it with "Can"

Reply:
added CAN into display name